### PR TITLE
remove bearerproxy

### PR DESCRIPTION
**Pull request by jr0d**
* Removes bearer proxy
* Adds kubernetes-dashboard chart for 2.0beta6
* Modifies opsportal chart to allow the creation of an admin cluster role binding
* TFA now uses a dedicated serviceaccount with impersonation privileges

### CODE REVIEW

**Comment by jaxesn** [comment]
What is the plan here, like what do we do while we wait for upstream to merge/release?  Are we going to fork for a bit?

**Comment by jr0d** [comment]
@jaxesn I am hoping it gets reviewed and approved over the weekend, monday at the latest. If it takes longer, we can always just roll an image and pull it from mesosphere/kubernetes-dashboad.

File: staging/traefik-forward-auth/Chart.yaml [hunk 1]
```diff
@@ -3,12 +3,14 @@ apiVersion: v1
...
+version: 0.2.9
 keywords:
   - traefik-forward-auth
   - traefik
   - oauth
 home: https://github.com/mesosphere/traefik-forward-auth
 maintainers:
   - name: jongiddy
-    email: jgiddy@mesosphere.com
+    email: jgiddy@md2iq.com
```
Comment (joejulian) [inline staging/traefik-forward-auth/Chart.yaml]: typo

File: stable/opsportal/templates/hooks-roles.yaml [hunk 2]
```diff
@@ -1,43 +1,61 @@
...
+# running an elevated job that can create the ops-portal super user
 {{- if .Values.secrets.create }}
 ---
-apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
+apiVersion: v1
+kind: ServiceAccount
 metadata:
-  name: {{ template "opsportal.fullname" . }}-secrets
+  name: "{{.Release.Name}}-service-account-pre-install"
```
Comment (joejulian) [inline stable/opsportal/templates/hooks-roles.yaml]: This runs a risk of being too long. Consider defining a template in _helpers.tpl to generate the name the same way fullname is generated.

Comment (jr0d) [reply stable/opsportal/templates/hooks-roles.yaml]: Good catch, I'll go back to using the truncated name in the template.

Comment (jr0d) [reply stable/opsportal/templates/hooks-roles.yaml]: For some reason, changing this to `{{ template "opsportal.fullname" . }}-admin` breaks the chart.

Comment (joejulian) [reply stable/opsportal/templates/hooks-roles.yaml]: I'll offer a patch when I get to a computer.

**Pull request merged**